js/stringProcessing/matchParagraphs.js

```diff
@@ -1,26 +1,43 @@
-var regexMatch = require( "../stringProcessing/matchStringWithRegex.js" );
+var map = require( "lodash/map" );
+
+/**
+ * Matches the paragraphs in <p>-tags and returns the text in them.
+ * @param {string} text The text to match paragraph in.
+ * @returns {array} An array containing all paragraphs texts.
+ */
+var getParagraphsInPTags = function ( text ) {
```

Comment:
Please rename this into `getParagraphsInParagraphTags` or `getParagraphsInTags`.